utils/R/utils.R

```diff
@@ -665,6 +665,25 @@ convert.expr <- function(expression) {
   return(list(variable.drv = deri.var, variable.eqn = list(variables = variables, expression = deri.eqn)))
 }
 
+##' Simple function to use ncftpget for FTP downloads behind a firewall.
+##' Requires ncftpget and a properly formatted config file in the users 
+##' home directory
+##' @title download.file
+##' @param url complete URL for file download
+##' @param destfile destination file name
+##' @param method Method of file retrieval.  Currently the only option is ncftpget
+##' @export
+##' @author Shawn Serbin
+download.file <- function(url, destfile, method="ncftpget") {
+   if (method=="ncftpget") {
```

Comment:
@robkooper but where would we then set method?  in the download.XX.R files?  That is at the function call?

I could add a switch that if the ftplink (or similar) link has ftp:// the use download.ftp.method